include/InlineEditing/inlineEditing.js

```diff
@@ -1,524 +1,611 @@
-            'id': id,
-            'view' : view,
-            'to_pdf': true
-        }
-    );
-    $.ajaxSetup({"async": true});
-     if(result.responseText){
-         try {
-             return (JSON.parse(result.responseText));
-         } catch(e) {
-             return false;
-         }
-
-     }else{
-         return false;
-     }
-
-
-}
-
-/**
- * Ajax call retrieve the field value from the bean used for closing the input.
- * Calls a controller action in /modules/Home/controller.
- * Returns the formatted output value of the field.
- * @param field
- * @param module
- * @param id
- * @returns {*}
- */
-
-function loadFieldHTMLValue(field,id,module) {
-    $.ajaxSetup({"async": false});
-    var result = $.getJSON('index.php',
-        {
-            'module': 'Home',
-            'action': 'getDisplayValue',
-            'field': field,
-            'current_module': module,
-            'view': view,
-            'id': id,
-            'to_pdf': true
-        }
-    );
-    $.ajaxSetup({"async": true});
-
-    return(result.responseText);
-}
-
-/**
- * Ajax call to retrieve the field validation js this needs to be done separately as you can't json_encode javascript.
- * Calls a controller action in /modules/Home/controller.
- * Returns the add to validate call for the field..
- * @param field
- * @param module
- * @param id
- * @returns {*}
- */
-
-function getValidationRules(field,module,id){
-    $.ajaxSetup({"async": false});
-    var result = $.getJSON('index.php',
-        {
-            'module': 'Home',
-            'action': 'getValidationRules',
-            'field': field,
-            'current_module': module,
-            'id': id,
-            'to_pdf': true
-        }
-    );
-    $.ajaxSetup({"async": true});
-
-    try {
-        var validation = JSON.parse(result.responseText);
-    } catch(e) {
-        alert(SUGAR.language.translate('app_strings', 'LBL_LOADING_ERROR_INLINE_EDITING'));
-        return false;
-    }
-
-    return "<script type='text/javascript'>addToValidate('EditView', \"" + field + "\", \"" + validation['type'] + "\", " + validation['required'] + ",\"" + validation['label'] + "\");</script>";
-}
-
-/**
- * Ajax call to retrieve js needed for relate fields..
- * Calls a controller action in /modules/Home/controller.
- * Returns the javascript.
- * @param field
- * @param module
- * @param id
- * @returns {*}
- */
-
-function getRelateFieldJS(field, module, id){
-    $.ajaxSetup({"async": false});
-    var result = $.getJSON('index.php',
-        {
-            'module': 'Home',
-            'action': 'getRelateFieldJS',
-            'field': field,
-            'current_module': module,
-            'id': id,
-            'to_pdf': true
-        }
-    );
-    $.ajaxSetup({"async": true});
-
-    SUGAR.util.evalScript(result.responseText);
-
-    return result.responseText;
-}
+/*********************************************************************************

```

Comment:
there are so many differences because latest commit contains the commits pulled from salesagility/master branch.